AzerothAuctionTarget.py

```
@@ -162,7 +162,7 @@ def run(self):
             return
 
         marketshare_recommendations = requests.post(
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)